modules/custom-post-types/testimonial.php

```diff
@@ -585,6 +588,13 @@ static function get_testimonial_class( $testimonial_index_number, $columns ) {
 	 */
 	static function get_testimonial_thumbnail_link( $post_id ) {
 		if ( has_post_thumbnail( $post_id ) ) {
+			/**
+			 * Change the thumbnail size for the Testimonial CPT.
+			 *
+			 * @since 3.4.0
+			 *
+			 * @param string $var Size of thumbnail.
```

Comment:
Perhaps instead of size of thumbnail, indicate image size keyword. Do you know it it'll accept the `array(32, 32)` style? If both, may be helpful to indicate that.